variables.tf

```diff
@@ -68,3 +68,12 @@ variable "enable_classiclink_dns_support" {
   default     = false
 }
 
+variable "enable_custom_default_security_group" {
```

Comment:
```suggestion
variable "enable_default_security_group" {
```

why `custom`?

Reply:
Because the default is already created when is created the VPC, and the rules (ingress and agress "closed") are not equals to the defaults, which are:
- ingress:
-- allow all traffic from itself (SG)
- egress:
-- allow all traffic to all

Anyway I agree that "custom" is a bit misleading

Reply:
PS: I think that the even more accurate name would be enable_default_security_group_with_custom_rules but my fingers are already tired from writing it :-D